content/02.body.md

```diff
@@ -147,5 +148,28 @@ The edge prior was the best calibrated feature for both tasks.
 
 ## Discussion
 
+### Prediction performance and the edge prior
+
+We focus on (biomedical) network edge prediction.
+In particular, we are interested in predicting new edges with specificity, so that predictions reflect particular connectivity rather than generic node characteristics.
```

Comment:
```suggestion
Our overall goal is to predict new edges with specificity, so that predictions reflect particular connectivity rather than generic node characteristics.
```